tagstore/backends/django.py

```diff
@@ -94,7 +94,7 @@ def get_tags_in_taxonomy_hierarchically_as_dict(self, taxonomy_uid: TaxonomyId)
         taxonomy_uid_as_int = int(taxonomy_uid)
         for tag in TagModel.objects.filter(taxonomy_id=taxonomy_uid_as_int).order_by('path'):
             node = {'name': tag.name, 'id': tag.id, 'children': []}
-            as_tuple = Tag(taxonomy_uid=taxonomy_uid_as_int, name=tag.name)
+            as_tuple = Tag(taxonomy_uid=taxonomy_uid_as_int, name=tag.name.lower())
```

Comment:
This fixes the error in the admin, but not the root cause. The root cause is that `parent_tag_tuple` returns the wrong case in the tuple - it should return the correct capitalization of the tag, but because it reads `name` out of the `path` property, it's converted to lowercase.

I think the best fix is actually to change `make_path` (and the comment above the `path` field in the model) so that the materialized path is saved with the correct case of each tag instead of being in lowercase. I don't think we actually need it saved in lowercase, since we're using lowercase collation in MySQL.

Reply:
That makes good sense. I tried changing it in `make_path` but it caused `test_case_sensitive_tags` to fail with `ValueError: Tag testing already exists with a different parent tag.`. I will poke at it some more tomorrow.

Reply:
Okay, this might be closer to what you had in mind. I've taken `lower()` out of `make_path`, so tags are saved in their original case. The upshot of this was that in `_add_tag_to_taxonomy` it is necessary to explicitly lowercase the path when testing for tags with multiple parents. (Because '5:BioTech' != '5:biotech' although they are the same tag.)

Reply:
Thanks, looks good!